lib/game.rb

```diff
@@ -39,9 +39,19 @@ def turn_loop
     @turn_count += 1 unless @winner
   end
 
+  def get_player_move
+    @inOut.select_move(@board.max_turns)
+  end
+
   def submit_move
-    player_input = @inOut.select_move(@board.max_turns)
-    @board.make_move(current_player.mark, row = (player_input - 1) / @board.board_size, column = (player_input - 1) % @board.board_size)
+    valid_move = false
+    until(valid_move)
+      player_input = @inOut.select_move(@board.max_turns)
+      row = (player_input - 1) / @board.board_size
```

Comment:
Who's responsibility is it to know that an array is zero based, and the user input is 1 based?